dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.ad.ssl/server.xml

```diff
@@ -3,87 +3,41 @@
-		sslEnabled="true"
+		activedFilters="actived"
 		searchTimeout="8m"
-		sslRef="LDAPSSLSettings" 
-		recursiveSearch="true" >
-      <ldapEntityType name="Group" searchFilter="(ObjectCategory=Group)">
-        <objectClass>group</objectClass>
-      </ldapEntityType>
-      <ldapEntityType name="OrgContainer">
-        <rdnProperty name="o" objectClass="organization"/>
-        <rdnProperty name="ou" objectClass="organizationalUnit"/>
-        <rdnProperty name="dc" objectClass="domain"/>
-        <rdnProperty name="cn" objectClass="container"/>
-        <objectClass>organization</objectClass>
-        <objectClass>organizationalUnit</objectClass>
-        <objectClass>domain</objectClass>
-        <objectClass>container</objectClass>
-      </ldapEntityType>
-      <ldapEntityType name="PersonAccount" searchFilter="(|(ObjectCategory=User)(ObjectClass=User))">
-        <objectClass>user</objectClass>
-      </ldapEntityType>
-      <groupProperties>
-        <memberAttribute name="member" dummyMember="uid=dummy" objectClass="group" scope="direct"/>
-      </groupProperties>
-      <attributeConfiguration>
-      	<attribute defaultValue="544" name="userAccountControl" entityType="PersonAccount" />
-      	<attribute name="samAccountName" propertyName="uid" entityType="PersonAccount" />
-        <attribute substituteWithValueOf="cn" name="samAccountName" entityType="Group" />
-        <attribute defaultValue="8" name="groupType" entityType="Group" />
-        <attribute name="unicodePwd" propertyName="password" syntax="unicodePwd"/>
-        <attribute name="userprincipalname" propertyName="kerberosId" entityType="PersonAccount" />
-        <propertiesNotSupported name="description"/>
-        <propertiesNotSupported name="jpegPhoto"/>
-        <propertiesNotSupported name="labeledURI"/>
-        <propertiesNotSupported name="carLicense"/>
-        <propertiesNotSupported name="pager"/>
-        <propertiesNotSupported name="roomNumber"/>
-        <propertiesNotSupported name="localityName"/>
-        <propertiesNotSupported name="stateOrProvinceName"/>
-        <propertiesNotSupported name="countryName"/>
-        <propertiesNotSupported name="employeeNumber"/>
-        <propertiesNotSupported name="employeeType"/>
-        <propertiesNotSupported name="businessCategory"/>
-        <propertiesNotSupported name="departmentNumber"/>
-        <propertiesNotSupported name="homeAddress"/>
-        <propertiesNotSupported name="businessAddress"/>
-      </attributeConfiguration>
-      <loginProperty>cn</loginProperty>
-      <contextPool enabled="true" initialSize="1" maxSize="0" timeout="0ms"
-          waitTime="3000ms" preferredSize="3"/>
-      <ldapCache>
-        <attributesCache size="4000" timeout="1200ms" enabled="true" sizeLimit="2000"/>
-        <searchResultsCache size="2000" timeout="600ms" enabled="true" resultsSizeLimit="1000"/>
-      </ldapCache>
-      <failoverServers name="failoverLdapServers">
```

Comment:
The failover servers may need to be removed, since we are only bringing up one of them.